packages/legend-extension-dsl-serializer/src/models/protocols/pure/DSLSerializer_PureProtocolProcessorPlugin.ts

```diff
@@ -135,22 +142,31 @@ export class DSLSerializer_PureProtocolProcessorPlugin
           );
           const element = getBinding(path, context.graph);
           element.schemaId = elementProtocol.schemaId;
-          element.schemaSet = elementProtocol.schemaSet;
-          element.contentType = guaranteeNonEmptyString(
-            elementProtocol.contentType,
-            `Binding 'contentType' field is missing or empty`,
+          if (elementProtocol.schemaSet !== undefined) {
+            element.schemaSet = V1_resolveSchemaSet(
```

Comment:
shouldn't we be creating `OptionalPackageableElementReference` here instead of `PackageableElementReference`?